src/platform/suecreek/include/platform/clk-map.h

```diff
@@ -31,14 +31,19 @@
 #ifndef __INCLUDE_CLOCK_MAP__
 #define __INCLUDE_CLOCK_MAP__
 
+#include <sof/clk.h>
+
 static const struct freq_table cpu_freq[] = {
 	{120000000, 120000, 0x0},
 	{400000000, 400000, 0x4}, /* default */
 };
 
+/* IMPORTANT: array should be filled in increasing order
+ * (regarding to .freq field)
+ */
 static const struct freq_table ssp_freq[] = {
-	{19200000, 19200, }, /* default */
-	{24000000, 24000, },
+	{ 38400000, 38400, CLOCK_SSP_XTAL_OSCILLATOR },
+	{ 96000000, 96000, CLOCK_SSP_PLL_FIXED },
```

Comment:
Are those values correct? It'd be really surprising that SueCreek has the 96kHz PLL that's typically used for HDaudio? Also the oscillator value should be checked.

Reply:
With @lbetlej we checked in specs that all CAVS platform support PLL Fixed Clock, so I add it here. We also got information that cavs fw uses 38.4 as XTAL freq.